### PR TITLE
Implement occlusion query

### PR DESCRIPTION
**Pull request by fyellin**
Implementation turned out to be straightforward.

### CODE REVIEW

File: tests/test_wgpu_occlusion_query.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+
+import wgpu
+from pytest import skip
+from testutils import run_tests, get_default_device
+from testutils import can_use_wgpu_lib, is_ci
+from wgpu import flags
+
+if not can_use_wgpu_lib:
+    skip("Skipping tests that need the wgpu lib", allow_module_level=True)
+elif is_ci and sys.platform == "win32":
```
Comment (Korijn) [inline tests/test_wgpu_occlusion_query.py:16]: Is it possible to make this conditional on DirectX being used?

**Comment by fyellin** [comment]
I don't have access to a Windows machine, so I'm not exactly sure what this
is supposed to be and I have no way to test it.

I copied these lines directly from `test_wgpu_native_render.py`.

File: tests/test_wgpu_occlusion_query.py (hunk 1 above)
Comment (almarklein) [reply tests/test_wgpu_occlusion_query.py:16]: At some point we should get back to these and see if they work now on CI. Also see #459

**Review by almarklein — APPROVED** [review]

**Comment by fyellin** [comment]
@almarklein .  You need to approve again.  Apparently because there was one unresolved conversation at the time you approved, it didn't count.

**Comment by Korijn** [comment]
Thank you! We greatly appreciate your contributions.

**Comment by fyellin** [comment]
By the way, I've also got RenderBundles working, though I haven't yet
written any significant tests.  (I have rewritten some vanilla code to use
them, so I know the basic idea is sound.)

Is there some bulletin or forum I should use to announce this? How do I
find out information like "Which Python versions do we support"?

**Comment by almarklein** [comment]
> By the way, I've also got RenderBundles working,

Nice! 🚀 

> Is there some bulletin or forum I should use to announce this?

There's the release notes where new features are mentioned.

There's also https://github.com/pygfx/wgpu-py/discussions but it is not used much, to be honest. Maybe it'd be nice to have a monthly newsletter of sorts 🤔

**Pull request merged**